.github/workflows/ios-signing-validate.yml

```
@@ -21,27 +20,32 @@ on:
-env:
-  WORKING_DIR: ios
-  KEYCHAIN_NAME: build-${{ github.run_id }}.keychain-db
-  KEYCHAIN_PATH: ${{ runner.temp }}/${{ env.KEYCHAIN_NAME }}
-
 jobs:
   validate-signing:
     runs-on: macos-15
+    env:
+      WORKING_DIR: ios
+      KEYCHAIN_NAME: build-${{ github.run_id }}.keychain-db
+      KEYCHAIN_PATH: ${{ runner.temp }}/build-${{ github.run_id }}.keychain-db
```

actionlint (1.7.7)

context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)